src/Prism.Core/Mvvm/ViewModelLocationProvider.cs

```diff
@@ -95,6 +97,9 @@ public static void SetDefaultViewTypeToViewModelTypeResolver(Func<Type, Type> vi
             _defaultViewTypeToViewModelTypeResolver = viewTypeToViewModelTypeResolver;
         }
 
+        public static void SetDefaultViewToViewModelTypeResolver(Func<object, Type> viewToViewModelTypeResolver) =>
```

Comment:
This looks good, just add some XML comments so we can clarify the different between this method and the existing SetDefaultViewType....